wp-includes/widgets/class-wp-widget-media-audio.php

```diff
@@ -151,7 +151,7 @@ public function enqueue_admin_scripts() {
 
 		$exported_schema = array();
 		foreach ( $this->get_instance_schema() as $field => $field_schema ) {
-			$exported_schema[ $field ] = wp_array_slice_assoc( $field_schema, array( 'type', 'default', 'enum', 'minimum', 'format', 'media_prop' ) );
+			$exported_schema[ $field ] = wp_array_slice_assoc( $field_schema, array( 'type', 'default', 'enum', 'minimum', 'format', 'media_prop', 'should_preview_update' ) );
```

Comment:
Yeah I was thinking about that too. I mean since `should_preview_update` is part of the base schema too it would make sense to have that included by default. But it does add more complexity. Not sure